src/components/docs-version-switcher/docs-version-switcher.module.css

```diff
@@ -1,114 +1,36 @@
-.root {
-  --padding-left-right: 24px;
-
-  background-color: white;
-  width: 100%;
-
-  /* custom elevation; no HDS token at the time of building */
-  box-shadow: 0 -2px 2px rgba(101, 106, 118, 0.1),
-    0 -8px 16px -10px rgba(101, 106, 118, 0.1);
-
-  &:hover {
-    /* custom elevation; no HDS token at the time of building */
-    box-shadow: 0 -2px 4px rgba(101, 106, 118, 0.15),
-      0 -16px 18px -8px rgba(101, 106, 118, 0.1);
-  }
-}
-
-.root-expanded {
-  /* custom elevation; no HDS token at the time of building */
-  box-shadow: 0 -2px 4px rgba(101, 106, 118, 0.15),
-    0 -16px 18px -8px rgba(101, 106, 118, 0.1);
-}
-
-.activator {
-  background-color: white;
-  border: none;
-  color: var(--token-color-foreground-faint);
-  padding-bottom: 12px;
-  padding-left: var(--padding-left-right);
-  padding-right: var(--padding-left-right);
-  padding-top: 12px;
-
-  &:focus {
-    outline: transparent;
-  }
-
-  &:focus-visible {
-    & .activatorTextWrapper {
-      box-shadow: var(--token-focus-ring-action-box-shadow);
-    }
-  }
-}
-
-.activatorTextWrapper {
-  align-items: center;
-  border-radius: 5px;
-  display: flex;
-  gap: 6px;
-  padding: 8px;
-  width: 100%;
-  position: relative;
-}
-
-.activatorText {
-  flex-grow: 1;
-  text-align: left;
-}
-
-.activatorIcon {
-  flex-shrink: 0;
-  display: flex;
-}
-
-.content {
-  padding-left: calc(var(--padding-left-right) - 3px);
-  padding-right: calc(var(--padding-left-right) - 3px);
-}
-
-.optionList {
-  border-top: 1px solid var(--token-color-border-primary);
-  padding-top: 12px;
-  padding-bottom: 12px;
-  max-height: 200px;
-  overflow-x: hidden;
-  overflow-y: auto;
-}
-
-.optionListItem {
-  padding-left: 3px;
-  padding-right: 3px;
-}
-
-.optionLink {
-  /* composition */
-  composes: g-focus-ring-from-box-shadow from global;
-
-  /* css properties */
-  background-color: white;
-  border-radius: 5px;
-  color: var(--token-color-foreground-faint);
-  display: block;
-  font-family: var(--token-typography-body-200-font-family);
-  font-size: var(--token-typography-body-200-font-size);
-  line-height: var(--token-typography-body-200-line-height);
-  padding: 8px;
-  position: relative;
-  text-decoration: none;
-
-  &[aria-current='page'] {
-    background-color: var(--token-color-palette-neutral-200);
-    color: var(--token-color-foreground-strong);
-  }
-
-  &:hover {
-    background-color: var(--token-color-palette-neutral-100);
-    color: var(--token-color-foreground-strong);
-    text-decoration: none;
-  }
-
-  &:focus-visible {
-    color: var(--token-color-foreground-strong);
-    z-index: 1;
-  }
+.docsVersionSwitcher {
+	@media (--small) {
+		width: 100%;
+
+		& button {
+			width: 100%;
+			justify-content: space-between;
+		}
+
+		& > div {
+			width: 100%;
+		}
+
+		& ul {
+			width: 100%;
+			max-width: 100%;
+		}
+	}
+
+	& ul {
+		max-height: 234px;
+		overflow: scroll;
```

Comment:
suggestion: can we use `overflow: auto` here so that the scrollbars will only show up if necessary?
```suggestion
		overflow: auto;
```

Notice the unneeded horizontal scrollbar here:
![Screen Shot 2022-09-09 at 1 04 04 PM](https://user-images.githubusercontent.com/1289701/189415718-80ae15f9-cf52-4d43-bfb1-1d2a8e64e1cb.png)

Reply:
[Done here](https://github.com/hashicorp/dev-portal/pull/945/commits/f199daa4cffe74cea11b9e341d590a679d7b9efd#diff-d6b6fc60cb7fc4bc24f617ad293058ae0b7f76ced0f7605fa3d4db08dbbc69a0L22-R22) 😄